doc/mk_sdcard_image.sh

```diff
@@ -3,6 +3,7 @@
 MOUNT_POINT="/tmp/mntpoint"
 CUR_STEP=1
 DOMA_PART_N=p3
+DEFAULT_IMAGE_SIZE_GIB=7
```

Comment:
why 7? not 8 ;)

Reply:
Reworked.
DEFAULT_IMAGE_SIZE_GIB is defined inside partition_image() as sum of sizes of all partitions.